src/client/replication_ddl_client.cpp

```diff
@@ -1434,23 +1445,83 @@ bool replication_ddl_client::valid_app_char(int c)
     return (bool)std::isalnum(c) || c == '_' || c == '.' || c == ':';
 }
 
+namespace {
+
+inline bool is_busy(const dsn::error_code &err)
```

Comment:
inline bool is_busy(const dsn::error_code &err) const

Reply:
Well, actually this function is not a member of a class ~